CMakeLists.txt

```diff
@@ -44,17 +45,19 @@ if (WIN32)
 endif ()
 
 # Exceptions for external libraries
-option(SGCT_DEP_INCLUDE_FMT "Include FMT library" ON)

```

Comment:
We don't align the different arguments of subsequent function calls

Reply:
fixed